spec/user_interface_spec.rb

```diff
@@ -221,6 +220,67 @@
     end
   end
 
+  describe '#search_term' do
+    it 'prints a prompt asking user for a search term' do
+      input = StringIO.new("john\n")
+      ui = described_class.new(input, output, validator)
+
+      ui.search_term
+
+      expect(output.string).to include(described_class::SEARCH_MESSAGE)
+    end
+
+    it 'returns the search term typed by user' do
+      input = StringIO.new("john\n")
+      ui = described_class.new(input, output, validator)
+
+      expect(ui.search_term).to eq('john')
+    end
+
+    it 'prints error message if no input is given' do
+      input = StringIO.new("\njohn\n")
```

Comment:
according to this test description ("...if no input is given" ) you just need:

```ruby
input = StringIO.new("\n")
```
nope?